packages/styled-base/src/index.js

```diff
@@ -106,6 +106,17 @@ let createStyled: CreateStyled = (tag: any, options?: StyledOptions) => {
                 props.className
               )
             }
+            if (
```

Comment:
IMHO this should just be else if to the preceding if statement, like this:
```js
if (typeof props.className === 'string') {
  // ...
} else if (props.className != null) {
  className += props.className
}
```

Reply:
Ahhh yes that does make sense